examples/test_performance/REAME.md

```diff
@@ -0,0 +1,10 @@
+TEST PERFORMANCE
+==
+Script that generates a graph of how latency and throughput of an NF chain changes based on its length
+
+To be determined
+-fix graph, what axes and measurments to use
+-specify before running, what # NFs to use (user will input as a flag)
+
+cd examples/test_performance
+./test_performance.sh ###SPECIFY NUMBER OF NFS
```

Comment:
See other docs like https://github.com/sdnfv/openNetVM/blob/master/docs/Examples.md to see how we specify required and optional arguments

Put these lines in a code block (see https://guides.github.com/features/mastering-markdown/)

Reply:
Ok, I'm still unsure as to how to get the shell script to take input from the command line arguments, which was one of the issues I mentioned in the slack

Reply:
Here are some examples of ways to make a bash script process command line arguments https://www.redhat.com/sysadmin/arguments-options-bash-scripts